ee/tabby-db/src/repositories.rs

```diff
@@ -68,13 +83,13 @@ impl DbConn {
             .await?)
     }
 
-    pub async fn update_repository_url(&self, name: String, git_url: String) -> Result<bool> {
+    pub async fn update_repository_url(&self, id: i32, git_url: String) -> Result<bool> {
```

Comment:
```suggestion
    pub async fn update_repository(&self, id: i32, name: String, git_url: String) -> Result<()> {
```

Reply:
Do we want to have one function that updates both the name and URL? Or one for each? This one was intended to be just for updating the URL. Also, this function returns a boolean to indicate whether the value was successfully updated - do we want to discard that information?

Reply:
for DAO functions, i suggest by default just provide an update function with all necessary parameters. We can always add new functions in future.

> Also, this function returns a boolean to indicate whether the value was successfully updated - do we want to discard that information?

See another comments, return error when updated != 1